vtex/ab-tester
# Use periodic update to save data periodically on Elasticsearch

#### What is the purpose of this pull request?
Make the periodic update also save the granular data, as a time series, on Elasticsearch.

#### What problem is this solving?
Use Elasticsearch to store time series data.

#### How should this be manually tested?
Install this version in an account and verify that the respective Elasticsearch is being updated.

#### Types of changes
